scripts/build.sh

```diff
@@ -28,21 +28,34 @@ __proj_dir="$(dirname "$__dir")"
 # shellcheck source=scripts/common.sh
```

Comment:
Changes in this file will be overwritten by https://github.com/intelsdi-x/snap-pluginsync unless you mark this as unmanaged in `.sync.yml`